rialto_airflow/cleanup.py

```diff
@@ -13,11 +14,17 @@ def cleanup_author_files(cleanup_interval_days: int, data_dir: str):
         Path(data_dir).glob("authors_active.csv.*")
     )
     for file in files:
-        file_time = datetime.strptime(file.name.split(".")[-1], "%Y-%m-%d")
-        age = current_time - file_time
-        if age.days > cleanup_interval_days:
-            logging.info(f"Removing author file: {file} (age: {age.days} days)")
-            file.unlink()
+        date_str = file.name.split(".")[-1]
+        if re.fullmatch(r"\d{2}-\d{2}-\d{4}", date_str):
+            file_time = datetime.strptime(date_str, "%m-%d-%Y")
```

Comment:
the actual fix is here from line 16 above, the format of these files is 05-25-2025 and not 2025-05-25

the rest of the code is some verification via regex that we have something that looks like a date